includes/class-newspack-newsletters.php

```diff
@@ -129,6 +129,8 @@ public static function remove_other_editor_modifications() {
 
 		remove_editor_styles();
 		remove_theme_support( 'editor-color-palette' );
+		add_theme_support( 'editor-gradient-presets', array() );
+		add_theme_support( 'disable-custom-gradients' );
```

Comment:
This introduces a duplication, there's already a `disable_gradients` method on this class – it should be removed (along with the `add_action` call which triggers it).

Reply:
I've removed this and changed the action for `disable_gradients` in https://github.com/Automattic/newspack-newsletters/pull/152/commits/3e16e6f1bd407b8d6f9394a0d2ecf3b19da054d2